### DISCUSSION TITLE
How old is the CEO of ⬇️🏢?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1101] [Galaxy Trivia] [answered]
```
What is greater than God,
more evil than the devil,
the poor have it,
the rich need it,
and if you eat it, you'll die?
```

**Comment by peterbaccaro** [comment]
The Eisenhower Executive Office Building (EEOB), located in Washington, D.C., was completed in 1888. It serves as an office building for various departments and agencies within the U.S. government, including the Office of the Vice President. As of 2023, the building is **134 years old**. It was named after President Dwight D. Eisenhower and is a historic and significant structure within the U.S. government complex.

The answer to the riddle is **nothing**.

**Comment by sheri-na** [comment]
Nothing is the answer.
Nothing is greater than God,
Nothing is more evil than the devil,
the poor have nothing,
the rich need nothing,
and if you eat nothing, you'll die.

**Comment by ParanoidUser** [comment]
**Hint:** Discussion topic and puzzle are related. 🔗

**Reply by peterbaccaro** [reply]
The CEO of the Nothing brand Carl Pei was born on the 11th September 1989 so that makes him **33 years old**.

**Comment by thuongtruong1009** [comment, marked as answer]
As of my last knowledge update in September 2021, I don't have real-time data on the current age of specific individuals. The CEO of the company represented by ⬇️🏢 could have changed since then. Please provide the name of the company, and I can help you look up the current CEO's age if that information is publicly available.

Regarding the riddle, the answer is "nothing." The riddle describes the concept of "nothing" in a poetic and paradoxical manner, highlighting its unique characteristics.

**Comment by arnob016** [comment]
**Nothing** is greater than God,
more evil than the devil,
the poor have it,
the rich need it,
and if you eat it, you'll die?

So  How old is the CEO of **Nothing**  .
That's Carl Pei. He's born at 1989 , so he's 33 years old now.
Answer : 33 years old.
